app/assets/javascripts/controllers/alerts/alerts_overview_controller.js

```diff
@@ -236,7 +235,9 @@ angular.module('ManageIQ').controller('alertsOverviewController',
           vm.displayFilter = alertsCenterService.displayFilters[0];
         }
 
-        vm.displayFilters = alertsCenterService.displayFilters;
+        vm.displayFilters = alertsCenterService.displayFilters.map(item => {
```

Comment:
I'm afraid we cannot use that fancy-schmancy `=>` here, am I correct @himdel ?

Reply:
Yes, that's likely the reason for travis failing.

`app/assets/javascripts` is ES5 only.